### ISSUE TITLE
lots of KeyError:NUMBER while running pytest

### ISSUE THREAD

**Issue by Trophime** [CalebBell/fluids#36]
Trying to build from scratch latest release on Debian/Testing with python 3.8
I had some issues with running the tests with python3.8 -m pytest tests:

```
============================= test session starts ==============================
platform linux -- Python 3.8.6, pytest-4.6.11, py-1.9.0, pluggy-0.13.0
rootdir: /build/fluids-0.1.86, inifile: pytest.ini
plugins: cov-2.8.1
collected 558 items / 35 errors / 523 selected

==================================== ERRORS ====================================
____ ERROR collecting .pybuild/cpython3_3.8_fluids/build/tests/__init__.py _____
/usr/lib/python3/dist-packages/_pytest/runner.py:226: in from_call
    result = func()
/usr/lib/python3/dist-packages/_pytest/runner.py:253: in <lambda>
    call = CallInfo.from_call(lambda: list(collector.collect()), "collect")
/usr/lib/python3/dist-packages/_pytest/doctest.py:432: in collect
    optionflags = get_optionflags(self)
/usr/lib/python3/dist-packages/_pytest/doctest.py:302: in get_optionflags
    flag_acc |= flag_lookup_table[flag]
E   KeyError: 'NUMBER'
_ ERROR collecting .pybuild/cpython3_3.8_fluids/build/tests/test_atmosphere.py _
/usr/lib/python3/dist-packages/_pytest/runner.py:226: in from_call
    result = func()
/usr/lib/python3/dist-packages/_pytest/runner.py:253: in <lambda>
    call = CallInfo.from_call(lambda: list(collector.collect()), "collect")
/usr/lib/python3/dist-packages/_pytest/doctest.py:432: in collect
    optionflags = get_optionflags(self)
/usr/lib/python3/dist-packages/_pytest/doctest.py:302: in get_optionflags
    flag_acc |= flag_lookup_table[flag]
E   KeyError: 'NUMBER'
...
```

Any hint to solve this problem would be great.
Thanks

**Comment by CalebBell** [comment]
Hi Dr. Trophime,

NUMBER is a flag that allows for doctests to be more stable across different platforms. It was added in pytest 5.1.0 (2019-08-15) which is less than the current pytest version in debian. 
I reverted the change for now so it should build for you again. I may try to re-enable it again later but only for versions of pytest which are sufficiently new as to support the feature :)

Sincerely,
Caleb
